develop/toolkit/api-libraries/polkadot-js-sdk.md

```diff
@@ -1,4 +1,112 @@
 ---
-title: Polkadot.js SDK
-description: TODO
----
\ No newline at end of file
+title: Polkadot.js API
+description: Interact with Polkadot SDK-based chains easily using the Polkadot.js API. Query chain data, submit transactions, and more via JavaScript.
```

Comment:
Since in the introduction, it has been added the term `Javascript/Typescript`, I think it makes sense to keep this description consistent with that

Note: this desc has 151 chars 😃 
```suggestion
description: Interact with Polkadot SDK-based chains easily using the Polkadot.js API. Query chain data, submit transactions, and more via JavaScript or Typescript.
```